commitrmnocb/report.go

```diff
@@ -0,0 +1,84 @@
+package commitrmnocb
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+
+	"github.com/smartcontractkit/chainlink-ccip/commit"
+)
+
+func (p *Plugin) Reports(seqNr uint64, outcomeBytes ocr3types.Outcome) ([]ocr3types.ReportWithInfo[[]byte], error) {
+	outcome, err := DecodeCommitPluginOutcome(outcomeBytes)
+	if err != nil {
+		// TODO: metrics
+		p.lggr.Errorw("failed to decode Outcome", "outcomeBytes", outcomeBytes, "err", err)
+		return nil, fmt.Errorf("failed to decode Outcome: %w", err)
+	}
+
+	if outcome.OutcomeType != ReportGenerated {
+		return []ocr3types.ReportWithInfo[[]byte]{}, nil
+	}
```

Comment:
Worth adding a log for this case?

Reply:
This is the happy path (e.g. in the round when intervals are chosen), so I think logging here would just be verbose.